src/h.js

```diff
@@ -28,4 +28,3 @@ export default function(tag, data) {
       }
     : tag(data, children)
 }
-
```

Comment:
This tiny change somehow doesn't conform to https://github.com/hyperapp/hyperapp/blob/master/.editorconfig#L7 imo 🐰